build.gradle

```diff
@@ -1,21 +1,13 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
-buildscript {
-    ext.apollo_version = '2.5.3'
-
-    repositories {
-        google()
-        mavenCentral()
-    }
-    dependencies {
-        classpath "com.android.tools.build:gradle:7.0.2"
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.20"
-        classpath "com.apollographql.apollo:apollo-gradle-plugin:$apollo_version"
-
-        // NOTE: Do not place your application dependencies here; they belong
-        // in the individual module build.gradle files
-    }
+plugins {
+    id 'com.android.application' version '7.1.1' apply false
+    id 'com.android.library' version '7.1.1' apply false
+    id 'org.jetbrains.kotlin.android' version '1.6.10' apply false
+    id 'io.github.gradle-nexus.publish-plugin' version '1.1.0'
+    id 'com.apollographql.apollo' version '2.5.12' apply false
```

Comment:
Can we shift the `ext { APOLLO_VERSION = '2.5.12' }` block into this file, so that it can be used in the root build.gradle as well as the app build.gradle file?